templates/taxi/car_detail.html

```diff
@@ -10,6 +10,15 @@ <h1>
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
+    {% if user not in car.drivers.all %}
+      <a href="{% url 'taxi:assign-driver' pk=car.id %}" class="btn btn-primary link-to-page">
+        Assign me to this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:remove-driver' pk=car.id %}" class="btn btn-primary link-to-page">
+        Delete me from this car
+      </a>
+    {% endif %}
   </h1>
```

Comment:
Code Style issue: There is no need for the `Delete`, `Update` and `Assign me to this car` buttons to be inside the `<h1>` tag. It's better to put them outside of it.